charts/nri-metadata-injection/tests/affinity_test.yaml

```diff
@@ -0,0 +1,37 @@
+suite: test affinity setup
```

Comment:
Same as in the previous commit. I would not retest it.

Reply:
[a3d79f0](https://github.com/newrelic/k8s-metadata-injection/pull/163/commits/a3d79f01dfa8535d1c29c1535fe91eacbe949551)